docker/cla_frontend.ini

```diff
@@ -12,3 +12,5 @@ buffer-size=32768
 post-buffering-bufsize=32768
 http-workers=5
 alarm=removefile cmd:rm /tmp/listen_queue_healthy
+alarm-backlog=removefile cmd:rm /tmp/listen_queue_healthy
```

Comment:
I think we just need the alarm name here alarm-backlog

Reply:
amended